src/renderer/features/governance/aggregates/voting.ts

```diff
@@ -34,6 +34,28 @@ const $activeWalletVotes = combine(
   },
 );
 
+const $possibleAccountsForVoting = combine(
+  walletModel.$activeWallet,
+  networkSelectorModel.$governanceChain,
+  (wallet, chain) => {
+    if (nullable(wallet) || nullable(chain)) return [];
+
+    if (walletUtils.isPolkadotVault(wallet)) {
+      const shards = wallet.accounts.filter((a) => accountUtils.isShardAccount(a) && a.chainId === chain.chainId);
+
+      if (shards.length) {
+        return shards;
+      } else {
+        return wallet.accounts.filter(
+          (a) => accountUtils.isBaseAccount(a) && accountUtils.isChainAndCryptoMatch(a, chain),
+        );
+      }
```

Comment:
PV wallet can includes not only shard account. Here we may lost chain accounts. And if we don't have shards can use base account, which should be excluded.

Reply:
In case of voting mutishard accounts can vote only with shards, not base account
CC @stepanLav